swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition = None):
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = 0 if condition == None else condition
+
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
```

Comment:
Here, you hardcoded the class type with "Item". Why not reuse your `get_category` method so that you can get the class name from any instance that inherits from the Item class. 

```python
return f"An object of type {self.get_category()} with id {self.id}."